### PR TITLE
your codes are so fucking ugly

### PR DESCRIPTION
**Pull request by Weebifying**
title

### CODE REVIEW

**Comment by coopeeo** [comment]
its not even our fault
most of the time github fucks it up

**Comment by Weebifying** [comment]
how come its github's fault :sob:

**Comment by Weebifying** [comment]
also i replaced all 2-space tabs with 2 spaces so the indents should look good now

**Comment by coopeeo** [comment]
cause it want to fucking change it every damn time i commit from vscode (github.dev)
klajglk;jfaj;lkgfrop;hwpoerhj

File: src/main.cpp [hunk 1]
```diff
@@ -185,87 +185,89 @@ class $modify(PlayerObject) {
 // i was getting mad at the function, (i'm bi i just think it's funni - viper)
 class $modify(YoureAPansexual,PauseLayer) {
   void YOUAREGAY(CCObject*) {
-  		geode::openSettingsPopup(Mod::get());
+      geode::openSettingsPopup(Mod::get());
   }
 
+  // fuck yall i fixed the indentation for this snippet
```
Comment (coopeeo) [inline src/main.cpp:191]: no remove idk sure idk why not

**Review by BeatACVR — APPROVED** [review]
fuck you too >/:3

**Comment by Weebifying** [comment]
id rather use notepad than github.dev tbh

**Review by coopeeo — CHANGES_REQUESTED** [review]
remove rude comment against devs.
this is also why i said oh no in #github channel

**Comment by coopeeo** [comment]
DAMN IT
@BeatACVR WHY

File: src/main.cpp
Comment (coopeeo) [inline src/main.cpp:1]: we goog

**Review by coopeeo — COMMENTED** [review]
eeee

**Review by coopeeo — COMMENTED** [review]
FUCK ME

**Comment by coopeeo** [comment]
LET ME FUCKING CHANGE MY REVIEW GITTY HUBBY

**Comment by Weebifying** [comment]
what the hell is happening

**Comment by coopeeo** [comment]
there we go better title

**Comment by Weebifying** [comment]
this is abuse

**Pull request merged**